src/components/Tree/Tree.tsx

```diff
@@ -10,20 +24,27 @@ import type {
     TreeState,
     TreeStateAction,
 } from '@components/Tree/types';
-import { TreeContext } from '@components/Tree/TreeContext';
-import { DndWrapper, DraggableItem, DropZonePosition } from '@utilities/dnd';
-import { DEFAULT_TREE_ITEM_PADDING, cloneThroughFragments, flattenChildren } from './utils';
+import { TreeContext, TreeContextProps } from '@components/Tree/TreeContext';
+
+import { cloneThroughFragments, flattenChildren, fondueCollisionDetection, snapToHandle } from './utils';
+import { type Overlay, TreeItemOverlay } from './TreeItemOverlay';
 
-const noop = () => undefined;
 export const ROOT_ID = '__ROOT__';
 
+const measuring: MeasuringConfiguration = {
+    droppable: {
+        strategy: MeasuringStrategy.Always,
+    },
+};
+
 const reducer = (state: TreeState, action: TreeStateAction): TreeState => {
     switch (action.type) {
         case 'SET_SELECT': {
             const { id, isSelected } = action.payload;
 
             if (!state.items.has(id)) {
-                throw new Error(`No tree item registered with id ${id}.`);
+                console.error(`No tree item registered with id ${id}.`);
+                return state;
```

Comment:
It shouldn't be the case, if so there is an error introduced

Reply:
"This is the same thing as before, but it is failing gracefully/silently. This should be the case because if we throw an error, we will break the application. And that is a no-no to the end user."